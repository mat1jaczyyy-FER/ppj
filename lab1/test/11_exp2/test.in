(((((((a+b)-c)+d)/11)*34)*1234)-345)
