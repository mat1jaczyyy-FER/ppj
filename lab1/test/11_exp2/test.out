L_ZAGRADA 1 (
L_ZAGRADA 1 (
L_ZAGRADA 1 (
L_ZAGRADA 1 (
L_ZAGRADA 1 (
L_ZAGRADA 1 (
L_ZAGRADA 1 (
IDN 1 a
OP_PLUS 1 +
IDN 1 b
D_ZAGRADA 1 )
OP_MINUS 1 -
IDN 1 c
D_ZAGRADA 1 )
OP_PLUS 1 +
IDN 1 d
D_ZAGRADA 1 )
OP_DIJELI 1 /
BROJ 1 11
D_ZAGRADA 1 )
OP_PUTA 1 *
BROJ 1 34
D_ZAGRADA 1 )
OP_PUTA 1 *
BROJ 1 1234
D_ZAGRADA 1 )
OP_MINUS 1 -
BROJ 1 345
D_ZAGRADA 1 )
